src/components/Login/LoginContainer.tsx

```diff
@@ -160,8 +156,11 @@ export const InitialisationContainer: FunctionComponent<NavigationProps> = ({
       lastResendWarningMessageRef.current = response.warning ?? "";
       return true;
     } catch (e) {
-      if (e instanceof LoginError) {
-        showAlert({ ...e.alertProps, onOk: () => resetStage() });
+      if (e instanceof LoginLockedError) {
+        // TODO: To determine if this is the best way to show this
```

Comment:
What is this comment for?

Reply:
I wasn't sure if feeding a `Record<string, string>` into `showErrorAlert` in order to show dynamic portions in the dialog is a good idea, but it doesn't look like there's any other way. Do you have any other ideas?

Reply:
This was changed liao hor, so no need discuss le?

Reply:
Yes, this should work fine!